index.html

```diff
@@ -3,184 +3,196 @@
-            <h2 class="header">Recent posts</h2>
+<div  style="background-color: #eaeafa;">
+    <div class="width_capped">
+        <div class="dual_container">
+            <p class="dual_container_item" id="intro_blurb">Since 1976, Computer Science House has 
+                provided a revolutionary living and learning 
+                environment for its members. With unique 
+                facilities, an emphasis on hands-on learning, 
+                and a strong social atmosphere, CSH helps its 
+                members grow as professionals and more.</p>
+                <!-- TODO: Make the thing below a carousel with older group pictures -->
+            <img class="dual_container_item" src="assets/images/welcome_back_large.webp" alt="CSH stands together on a grass field"/>
+        </div>                
+    </div>                
+</div>
+
+<!-- Projects tab -->
+<h3 class="left_offset" style="padding-top: 0.6em;">Things we're working on ...</h3>
+<div class="quad_image">
+    <div class="content">
+        <a href="csh.rit.edu">
+            <div class="content-overlay"></div>
+            <img class="content-image" src="assets/images/server.png" style="width: 100%;" alt="A picture of CSH's server racks"/>
+            <div class="content-details fadeIn-bottom"">
+                <h3 class="content-title">Server room upgrade</h3>
+                <p>Done in collaboration with wayfair</p>
+            </div>
+        </a>
+    </div>
+    <div class="content">
+        <a href="csh.rit.edu">
+            <div class="content-overlay"></div>
+            <img class="content-image" src="assets/images/TUNES.png" style="width: 100%;" alt="TUNES, an old jukebox that's being made digital"/>
+            <div class="content-details fadeIn-bottom">
+                <h3 class="content-title">TUNES</h3>
+                <p>Modernizing a jukebox</p>
+            </div>
+        </a>
+    </div>
+    <div class="content">
+        <div class="content-overlay"></div>
+        <img class="content-image" src="assets/images/infosys.png" style="width: 100%;" alt="A LED matrix that makes up CSH's Infosys"/>
+        <div class="content-details fadeIn-bottom">
+            <h3 class="content-title">Infosys</h3>
+            <p>A legacy CSH project that is being modernized.</p>
         </div>
-        {% for post in site.posts limit:2 %}
-            {% include post.html post=post %}
-        {% endfor %}
-        <div class="col-12">
-            <a class="btn btn-primary" href="/blog/">More blog posts</a>
+    </div>
+    <div class="content">
+        <div class="content-overlay"></div>
+        <img class="content-image" src="assets/images/letmein.png" style="width: 100%;" alt="A raspberry pi and pencil sharpener on the wall, CSH's letmein system"/>
+        <div class="content-details fadeIn-bottom">
+            <h3 class="content-title">Letmein</h3>
+            <p>A legacy CSH project that is being modernized.</p>
```

Comment:
Could these be short descriptions?

Reply:
That was my thought. I decided on a short sentence because of the way it displays on both smaller desktop screens and on phones, there's not really enough space for a whole paragraph.